contrib/autopilot/autopilot.py

```diff
@@ -721,6 +721,8 @@ def report_data(file_sets):
 
     logging.info("=== Train translation model ===")
     logging.info("Model: %s", args.model)
+    if args.model == "gnmt_like":
+        logging.info("WARNING: This is an 8 layer LSTMs model which can resembles the 'GNMT' architecture.")
```

Comment:
This could be worded: `"NOTE: This is an 8 layer LSTM model similar (but not exactly identical) to the 'GNMT' architecture."`

Reply:
@mjdenkowski Thanks, changed:)